tree/ntuple/v7/src/RNTupleMerger.cxx

```diff
@@ -803,7 +803,10 @@ static void AddColumnsFromField(std::vector<RColumnMergeInfo> &columns, const RN
       const auto &srcColumn = srcDesc.GetColumnDescriptor(srcColumnId);
       RColumnMergeInfo info{};
       info.fColumnName = name + '.' + std::to_string(srcColumn.GetIndex());
-      info.fInputId = srcColumnId;
+      std::cerr << "Adding column " << info.fColumnName << "with id " << srcColumnId
+                << " (phys: " << srcColumn.GetPhysicalId() << ")\n";
```

Comment:
Is this debug output?

Reply:
Correct. I thought I removed it but apparently not from all git timelines. Thanks for spotting it